docs/variant_annotation.md

```diff
@@ -101,6 +101,18 @@ followed by `_vep_output.vcf`. Note that if this directory already exists, then
 Variant Transforms fails. This is to prevent unintentional overwriting of old
 annotated VCFs.
 
+* [`--shard_variants`](https://github.com/googlegenomics/gcp-variant-transforms/blob/master/gcp_variant_transforms/options/variant_transform_options.py#L290)
+by default, the input files are sharded into smaller temporary VCF files before
+running VEP annotation. If the input files are small, i.e., each VCF file
+contains less than 50,000 variants, set this flag to false can be more time
```

Comment:
nit: s/set/setting

I like the following slightly better, but leave it to you

", setting this flag can be computationally wasteful."